app/models/card.rb

```diff
@@ -55,14 +55,20 @@ def coordinates_are_both_set
   end
 
   def color
-    case card_type
+    color = case card_type
     when "youth" 
       "255,0,0"
     when "adult"
       "0,255,0"
     else 
       "128,128,128"
     end
+
+    if validity == "disabled"
+      color = "200,200,200"
+    end
+
+    color
   end
```

Comment:
I think this piece of code should be moved to an appriopriate place, either a view helper or directly in the javascript.